boskos/update_prow_config.sh

```diff
@@ -29,12 +29,14 @@ if [[ -a "${PROW_SERVICE_ACCOUNT:-}" ]] ; then
 fi
 
 if ! [ -x "$(command -v kubectl)" ]; then
-	gcloud components install kubectl 
+	gcloud components install kubectl
 fi
 
-pushd "${TREE}/boskos"
-make update-config
-popd
+
+# TODO(fejta): deploy this using the bazel target
+gcloud container clusters get-credentials --project=k8s-prow-builds --zone=us-central1-f prow
+kubectl create configmap resources --from-file=config=prow/cluster/boskos-resources.yaml --dry-run -o yaml | \
```

Comment:
This now has to be run from the root of k/t-i.
/shrug

Reply:
Oh lame. I thought it CD there already.